tt_metal/impl/program/program.hpp

```diff
@@ -151,7 +157,10 @@ class Program {
 
     bool is_finalized() const;
     bool is_cached() const;
+    ProgramBinaryStatus binaries_on_device_status(std::size_t device_id) const;
```

Comment:
It would be clearer to have this as:
`ProgramBinaryStatus get_program_binary_status(chip_id_t device_id) const;`
`ProgramBinaryStatus set_program_binary_status(chip_id_t device_id, ProgramBinaryStatus status);`

Reply:
Updated